src/components/consul/selectors/ImageSelector.js

```diff
@@ -175,6 +226,22 @@ const styles = StyleSheet.create({
     height: imageHeight(imageWidth() * 0.6),
     width: imageWidth() * 0.6
   },
+  sueDeleteImageButton: {
+    alignItems: 'center',
+    backgroundColor: colors.accent,
+    borderRadius: normalize(11),
+    height: normalize(22),
+    justifyContent: 'center',
+    left: normalize(10),
+    position: 'absolute',
+    top: normalize(10),
+    width: normalize(22),
+    zIndex: 5
+  },
+  sueImage: {
+    height: imageHeight(imageWidth() * 0.25),
+    width: imageWidth() * 0.3
```

Comment:
how come these sizes?

Reply:
The style code we previously used for the image preview in ImageSelector is as follows. Using this code as an example, I developed the style code for sue, but as far as I can see, this style doesn't always work properly. So I will update it.

```
  image: {
    height: imageHeight(imageWidth() * 0.6),
    width: imageWidth() * 0.6
  },
```

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c